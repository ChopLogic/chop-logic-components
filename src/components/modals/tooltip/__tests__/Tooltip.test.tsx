import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';
import ChopLogicTooltip from '../Tooltip';

describe('ChopLogicTooltip component', () => {
  const testTooltipProps = {
    tooltipContent: 'Test tooltip content',
    id: 'test-id',
  };

  it('ChopLogicTooltip should render the content', async () => {
    const { asFragment } = render(
      <ChopLogicTooltip {...testTooltipProps}>
        <span>Modal content</span>
      </ChopLogicTooltip>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
