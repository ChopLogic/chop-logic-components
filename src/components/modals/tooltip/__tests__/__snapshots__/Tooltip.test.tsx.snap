// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ChopLogicTooltip component > ChopLogicTooltip should render the content 1`] = `
<DocumentFragment>
  <span
    aria-describedby="test-id"
    class="_container_1d30ab"
    tabindex="0"
  >
    <span>
      Modal content
    </span>
  </span>
</DocumentFragment>
`;
